tests/test-cases/Group23-VIC-Machine-Service/23-03-VCH-Create.robot

```diff
@@ -134,7 +158,7 @@ Create complex VCH
     Verify Status Created
 
 
-    Inspect VCH %{VCH-NAME}-api-test-complex
+    Inspect VCH Config %{VCH-NAME}-api-test-complex
 
```

Comment:
Should we run `Verify VCH Initialization` for the `Create complex VCH` case as well?

Reply:
The purpose of `Create Complex VCH` is to check if all the VCH create params goes into VCH config, so some input params are fake just for testing purposes, like the `https-proxy` and `container-network` etc. So the VCH created by this test probably can't connect to docker endpoint...
Testing docker connectivity for the test case `Create minimal VCH within datacenter` should be sufficient, since the problem was with datacenter specification

Reply:
Thanks! It wasn't clear from the test whether this VCH would be functional.

Reply:
Yea...on API side we reused a lot of code and logic in CLI, so we assumed that if all the VCH create params on API successfully goes into VCH config, the VCH should be identically functional as created by CLI. That's why the API create integration test is not complete (for ex, no docker regression test)

Reply:
@AngieCris is it better to add some comment in the test.md file?

Reply:
@emlin Good point. Thanks!